test/val/bug1310.c

```diff
@@ -68,6 +68,21 @@ int main (void)
         failures++;
     }
 
+    if (-32768U != 32768U) {
+        fprintf (stderr, "Expected -32768U == 32768U, got: %ld\n", (long)-32768U);
+        failures++;
+    }
+    if (~32767U != 32768U) {
+        fprintf (stderr, "Expected ~32767U == 32768U, got: %ld\n", (long)~32767U);
+        failures++;
+    }
     printf ("failures: %u\n", failures);
+
+    if ((long*)0x1000 - (long*)0x2000 >= 0) {
+        fprintf (stderr, "Expected (long*)0x1000 - (long*)0x2000 < 0, got: %d\n", (long*)0x1000 - (long*)0x2000 >= 0);
```

Comment:
The third argument should be only the subtraction.  It shouldn't have the comparison.

Reply:
Fixed.